R/read_company_terms.R

```diff
@@ -1,29 +0,0 @@
-#' Read in company terms
-#'
-#' Function reads in company terms and checks for existence of required columns.
-#'
-#' @inheritParams run_trisk
-#' @param path Path to dir holding company terms data.
-#'
-#' @return A tibble holding company terms data or NULL
-read_company_terms <- function(path, use_company_terms) {
```

Comment:
I think if we drop read_company_terms everything related to use_company_terms (https://github.com/2DegreesInvesting/r2dii.climate.stress.test/blob/41f07ccf3254af18e7f0aa8fcb23ab98a2dbb874/R/run_trisk.R#L68) needs to disappear